yapapi/engine.py

```diff
@@ -448,26 +455,48 @@ async def _process_invoices(self) -> None:
             ):
                 break
 
+    async def _check_debit_note_rate(self, act_id: ActivityId, agr_id: AgreementId, job: "Job"):
```

Comment:
Why is this an `async` function when it does nothing `async`?

Reply:
It calls `await agreement.terminate`